packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -128,17 +128,30 @@ export enum EnqueueStrategy {
 
     /**
      * Matches any URLs that have the same hostname.
-     * For example, `https://wow.example.com/hello` will be matched for a base url of `https://example.com/`, but
+     * For example, `https://wow.example.com/hello` will be matched for a base url of `https://wow.example.com/`, but
      * `https://example.com/hello` will not be matched.
+     *
+     * @remark This mode will match `http` and `https` URLs, even if the base URL is `http` or `https`.
```

Comment:
i guess the `@remark` tag is not supported by the typedoc plugin, right? maybe it would be better to format it as a quote (`> ...`)? or is the tag somehow supported in vscode/webstorm?

also i'd call it "strategy" instead of "mode", and the sentence itself could deserve some improvement. maybe something like the following?

```
This strategy will match both `http` and `https` protocols regardless of the base URL protocol.
```